test/functional/rpc_validateaddress.py

```diff
@@ -109,21 +99,11 @@
         "Invalid Bech32 v0 address program size (16 bytes), per BIP141",
         [],
     ),
-    (
-        "tb1p0xlxvlhemja6c4dqv22uapctqupfhlxm9h8z3k2e72q4k9hcz7vq47Zagq",
-        "Invalid or unsupported Segwit (Bech32) or Base58 encoding.",  # tb1, Mixed case
-        [],
-    ),
     (
         "bc1p0xlxvlhemja6c4dqv22uapctqupfhlxm9h8z3k2e72q4k9hcz7v07qwwzcrf",
         "Invalid padding in Bech32 data section",  # zero padding of more than 4 bits
         [],
     ),
-    (
-        "tb1p0xlxvlhemja6c4dqv22uapctqupfhlxm9h8z3k2e72q4k9hcz7vpggkg4j",
-        "Invalid or unsupported Segwit (Bech32) or Base58 encoding.",  # tb1, Non-zero padding in 8-to-5 conversion
-        [],
-    ),
```

Comment:
could you please provide any explanation in the commit message why these were discarded?